het-sites.py

```diff
@@ -9,7 +9,9 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--homogen", type=float, default=1,
-                    help="cutoff fraction of base at site at which a site is considered to be homogenous.")
+                    help="Cutoff fraction of base at site at which a site is considered to be homogenous.")
```

Comment:
I always try to be precise about how a fraction is applied. E.g., if the fraction of the most common nucleotide at a site is at least this value, the site will be considered homogenous. By "precise" I mean to explicitly indicate whether the cutoff value is included or excluded. It helps the reader and it helps you when you go to put in a less-than or a less-than-or-equal, etc.

Reply:
Done